package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "package.json",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "task1": "nodemon ./src/task1.js",
```

Comment:
`nodemon` is fine for development, but it is better to launch an app with just `node ...` command :)